### ISSUE TITLE
Extract Falco endpoints loop

### ISSUE THREAD

**Issue by evilz** [pimbrouwers/Falco#45]
Hello,

I think it can be useful to extract this code 

https://github.com/pimbrouwers/Falco/blob/235a062a65457419eb3b7e2ff773846d0ada03de/src/Falco/Extensions.fs#L157-L170

in another extension method to do it manually and be more compatible with existing C# endpoint extension like this :

```fsharp
app
  .UseBlazorFrameworkFiles()
  .UseRouting()
  .UseEndpoints(fun e ->
                e.UseFalcoEndpoints(endpoints)   // <- new extension 
                e.UseHotReload() |> ignore
                e.MapBlazorHub() |> ignore
                e.MapFallbackToPage("/_Host") |> ignore)
```

If you ok I can also do this in a PR 😉

**Comment by pimbrouwers** [comment]
Hiya! Good idea. I will make this happen.

**Comment by pimbrouwers** [comment]
This has been completed in commit: https://github.com/pimbrouwers/Falco/commit/610d2044352a4ccfa917af69bd42a75f227afc7d and available in the 3.0.3 package. 

Thanks for the suggestion!
